.reuse/dep5

```diff
@@ -0,0 +1,122 @@
+License: Apache-2.0
+
+# Jon - see https://github.com/BinomialLLC/basis_universal/issues/165
+Files: lib/basisu/apg_bmp.*
+Copyright: 2019 Anton Gerdelan
+License: Apache-2.0
+
+Files: lib/basisu/basisu_astc_decomp.*
+Copyright: 2016 The Android Open Source Project
+License: Apache-2.0
+
+Files: lib/basisu/lodepng.*
+Copyright: 2005-2019 Lode Vandevenne
+License: Zlib
+
+Files: lib/basisu/*
+Copyright: 2019-2020 Binomial LLC
+License: Apache-2.0
+
+Files: cmake/modules/NSIS.InstallOptions.ini.in cmake/*.rtf
+Copyright: 2017-2020 Andreas Atteneder
+License: Apache-2.0
+
+Files: other_lib/*/*SDL* other_include/SDL2/*
+Copyright: 1997-2020 Sam Lantinga <slouken@libsdl.org>
+License: Zlib
+
+Files: other_lib/*/*/*zstd* other_include/zstd*
+Copyright: 2016-present Facebook, Inc.
+License: BSD-3-Clause
+
+Files: other_lib/mac/Release/libminizip.a
+Copyright: 1995-2017 Jean-loup Gailly and Mark Adler
+License: Zlib
+
+Files: other_include/assimp/* other_lib/*/*/*assimp* other_lib/mac/Release/libIrrXML.a
+Copyright: 2006-2019, assimp team
+License: BSD-3-Clause
+
+Files: other_include/GL/glew.h other_include/GL/glxew.h other_include/GL/wglew.h other_lib/win/Release-x65/glew32.*
+Copyright: 2008-2015, Nigel Stewart <nigels[]users sourceforge net>
+Copyright: 2002-2008, Milan Ikits <milan ikits[]ieee org>
+Copyright: 2002-2008, Marcelo E. Magallon <mmagallo[]debian org>
+Copyright: 2002, Lev Povalahev
+License: BSD-3-Clause
+
+Files: other_include/EGL/*.h other_include/GL*/*.h other_include/KHR/khrplatform.h
+Copyright: 2007-2020 The Khronos Group Inc.
+License: MIT
+
+Files: other_lib/win/*
+Copyright: 2020 Andreas Atteneder
+License: Apache-2.0
+
+Files: pkgdoc/*
+Copyright: 2018-2020 The Khronos Group Inc.
+License: Apache-2.0
+
+Files: tests/webgl/libktx-gltf/*
+Copyright: 2020 Don McCurdy, Austin Eng, Shrek Shao, and Mark Callow
+License: Apache-2.0
+
+Files: tests/webgl/libktx-webgl/*
+Copyright: 2020 Mark Callow
+License: Apache-2.0
+
+Files: tests/webgl/llt-three/*
+Copyright: 2020 Mark Callow
+License: Apache-2.0
+
+Files: tests/gtest/*
+Copyright: 2008, Google Inc.
+License: BSD-3-Clause
+
+Files: tools/package/mac/summary.rtfd/TXT.rtf
+Copyright: 2019 Mark Callow
+License: Apache-2.0
+
+Files: other_include/glm/*
+Copyright: 2005 G-Truc Creation
+License: MIT
+
+Files: lib/etcdec.cxx
+Copyright: 2013 Ericsson AB 2013. All Rights Reserved.
+License: LicenseRef-ETCSLA
+
+# This has an SPDX identifier but REUSE doesn't recognize it
+Files: tests/loadtests/vkloadtests/resources/ios/LaunchScreen.storyboard
+Copyright: 2017-2020 The Khronos Group Inc.
+License: Apache-2.0
+
+# These are resource files generated by Visual C++. Not sure what license
+# they have, if any. Assuming they share the main repo license for now.
+Files: tests/*/*loadtests.rc
+Copyright: 2020 The Khronos Group Inc.
+License: Apache-2.0
+
+# It's not certain that the stb font file indeed comes from tools in the
+# nothings/stb project on Github, but a reasonable assumption.
```

Comment:
Yes it does.

Reply:
I was unsure because the tool referred to in the header of the file no longer seems to exist in that repo, but the naming and purpose of the repo was consistent.